wallet-new/src/Cardano/Wallet/Kernel/Actions.hs

```diff
@@ -116,17 +118,23 @@ interp walletInterp action = do
 
       LogMessage txt -> emit txt
 
+      Shutdown -> return ()
```

Comment:
Let's bang an `error` here to signal this is an invariant violation. This is effectively dead code and the only reason why we need this is to please GHC.